frontend/src/scenes/userLogic.tsx

```diff
@@ -132,6 +133,15 @@ export const userLogic = kea<userLogicType<UserType, UpdateUserPayload>>({
         },
     }),
     events: ({ actions }) => ({
-        afterMount: [actions.loadUser],
+        afterMount: () => {
+            const preloadedUser = getAppContext()?.current_user
+            if (preloadedUser) {
+                actions.loadUserSuccess(preloadedUser)
+            } else if (preloadedUser === null) {
```

Comment:
For this to work well shouldn't you then default `getAppContext()` to return `undefined` instead of `null`? That way `null` means a logged out user, but `undefined` means there was an unhandled issue.

Reply:
`null?.anything` is also `undefined`, so it doesn't really matter if `getAppContext()` returns `null` or `undefined`. The user will only be `null` if explicitly set so in `getAppContext()`.

That said, I think it's fine for `getAppContext()` to actually return `undefined`. So changed.